tools/analyze_realm_metrics.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# This script will print metrics for the Realm library being deployed to end users.
+# To run it:
+#   1. Make sure that $D8 is defined in your environment, e.g. `D8="$ANDROID_SDK_ROOT/build-tools/29.0.2/d8"`
+#   2. Make sure that you have built the library artifacts using `./gradlew assembleRelease` from the realm folder.
+#   3. Run the script: `> sh ./analyze_realm_metrics.sh`
```

Comment:
BTW, running this from `tools` or `<root>` or `<root/realm>` always report `cat: ./unzippedBase/classes.dex: No such file or directory`

```
Analyzing Base...
AAR size: 2476366
cat: ./unzippedBase/classes.dex: No such file or directory
5641248 ./unzippedBase/jni/x86/librealm-jni.so

Analyzing ObjectServer...
AAR size: 3476405
cat: ./unzippedObjectServer/classes.dex: No such file or directory

```